packages/seacas/libraries/ioss/src/exodus/Ioex_DatabaseIO.C

```diff
@@ -648,6 +648,7 @@ namespace Ioex {
     int                 tstepCount     = 0;
     std::vector<double> tsteps(0);
 
+    // Use reference to make sure that no Region modifications occur based on the input flag
```

Comment:
I would say something like the `setRegionTimeSteps` flag determines whether we are actually populating the region timesteps or just querying the timesteps that are on a specific database without populating the regions timesteps data and setting the number of timesteps on the region... 

Maybe not much better than the coment above, but maybe says a little more about why  a client would want to do that...